Makefile

```diff
@@ -1,5 +1,14 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
 export RELEASE_BRANCH?=1-18
-export RELEASE?=1
+DEFAULT_RELEASE=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/RELEASE)
+ifneq ("$(PULL_NUMBER)","")
+	export RELEASE?=$(shell expr $(DEFAULT_RELEASE) \* 10000 + $(PULL_NUMBER))
```

Comment:
What are we trying to do here?

Reply:
This is generating a release number for a development build. In this case it would be 10137 since this is PR #137

Reply:
Why not make this the default for development? Is there a reason this can't just be configurable instead of forcing the user to use this logic?

Reply:
This isn't forcing the user to do anything, this is for Prow. PULL_NUMBER is a prow environment variable, although we are going to have to set it manually for postsubmits.